webapp/src/app/shared/table/table.component.html

```diff
@@ -88,7 +124,8 @@
                 >
                     <ng-template #cellDefinition let-column="column" let-columnWidth="columnWidth">
                         <mat-header-cell
-                            [style.flex-basis.%]="columnWidth"
+                        [style.flex-basis.px]="(screenWidth * columnWidth)"
```

Comment:
@sidharthjain-zemoso percent values were used to prevent table width flickering. What was the problem with them?